ouroboros-consensus/src/Ouroboros/Consensus/Util/STM.hs

```diff
@@ -38,7 +38,7 @@ import           Ouroboros.Consensus.Util.ResourceRegistry
   Misc
 -------------------------------------------------------------------------------}
 
--- | Wait until the TVar changed
+-- | Wait until a function of the TVar changed
```

Comment:
leftover? as it retries, it will wait until the TVars read by `getA` change, so I think it was correct previously